apps/web/src/components/HyloEditor/HyloEditor.js

```diff
@@ -169,6 +168,9 @@ const HyloEditor = React.forwardRef(({
 
   // Also update the useEffect to match the new onUpdate signature
   useEffect(() => {
+    if (editor.isInitialized) {
+      editor.commands.setContent(contentHTML)
```

Comment:
i think we now probably call onUpdate twice, once from the setContent, and once right below. should be able to remove the explicit call